src/index.html

```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
 
 		<!-- Facebook Meta Tags -->
-		<!-- TODO: <meta property="og:url" content="" />-->
+		<meta property="og:title" content="Antiplag">
+		<meta property="og:description" content="Check your files before submitting.">
+		<meta property="og:url" content="https://antiplag.sk/">		
 		<meta property="og:type" content="website" />
-		<!-- TODO: <meta property="og:title" content="" />-->
-		<!-- TODO: <meta property="og:description" content=""/>-->
 		<meta
 			property="og:image"
 			content="<%= htmlWebpackPlugin.files.favicon %>"
 		/>
 
 		<!-- Twitter Meta Tags -->
-		<!-- TODO: <meta property="twitter:url" content="" />-->
-		<meta name="twitter:card" content="summary_large_image" />
```

Comment:
Please leave this one here. It's obligatory from twitter's side:

https://css-tricks.com/essential-meta-tags-social-media/#proprietary-meta-tags